### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by loal007** [OpenNaja/cobra-tools#346]
helloi have this issue on blender
![blender cobra](https://github.com/OpenNaja/cobra-tools/assets/140377706/ecfd3654-b25f-4591-8130-f07c1d37cb6f)

**Comment by ilodev** [comment]
yeah, I was going to say the same.. adding the new success logging level on the config and not the logger setup makes all other tools error now:

ERROR:root:Could not create OVL: D:\steam\steamapps\common\Jurassic World Evolution 2\win64\ovldata\TestMod\Init.ovl

![image](https://github.com/OpenNaja/cobra-tools/assets/28944639/49dcfbb6-f380-4f0d-98cc-729e255f4136)

**Comment by ilodev** [comment]
for me adding these two lines worked.

```python

from ovl_util.config import logging_setup
logging_setup(__name__)
```

**Comment by hexabits** [comment]
> for me adding these two lines worked.
> 
> ```python
> from ovl_util.config import logging_setup
> logging_setup(__name__)
> ```
Yes, to use logging you have to setup logging.  The OP has a different issue.

**Comment by hexabits** [comment]
@loal007  Do you have more than one plugin installed and enabled at the same time?  Check your add-ons list.

**Comment by ilodev** [comment]
No, the plugging isn't calling logging_setup() anywhere

I think it is the same issue.

**Comment by hexabits** [comment]
> No, the plugging isn't calling logging_setup() anywhere
> 
> I think it is the same issue.

It is not the same issue.  Look at the exception message.  Also, logging_setup is called in `__init__.py` in the tools root.

**Comment by ilodev** [comment]
ah you are right, he has redefining issues, not undefined. My bad!!

**Comment by hexabits** [comment]
Also @ilodev you need to be using Python 3.11 now.

**Comment by hexabits** [comment]
> > Also @ilodev you need to be using Python 3.11 now.
> 
> :( oh, I'm conflicted with this now, two days ago 3.8 was still usable 🗡️

3.11 has been the recommended version for 5 months.

**Comment by ilodev** [comment]
Guess I’ll figure out what is not compatible with 3.8 because sublime gets
me stuck with this version for now

**Comment by hexabits** [comment]
@loal007  We confirmed ourselves that you have this issue because you have two copies of the add-on installed and enabled.  You will need to remove the duplicate.  If you update to latest there is now a better warning message telling you that you've installed or enabled it twice.

**Comment by loal007** [comment]
thanks for the answers
